examples/csv-io/csv_io.bal

```diff
@@ -46,15 +57,65 @@ function main(string... args) {
         log:printError("An error occurred while processing the records: ",
                         err = e);
     } finally {
-        //Close the text record channel.
-        match csvChannel.close() {
-            error sourceCloseError => {
-                log:printError("Error occured while closing the channel: ",
-                                err = sourceCloseError);
+        // Close the text record channel.
+        closeCSVChannel(csvChannel);
+    }
+
+    // Read the `.CSV` file as a table.
+    csvChannel = io:openCsvFile(srcFileName);
+    table<Salary> salaryTable;
+    try {
```

Comment:
Why do we need a try catch here? getTable method is returning errors

Reply:
Yeah I needed to confirm this. Wasn't really sure if there were cases that throw un-recoverable errors in I/O operations.